packages/lesswrong/components/admin/CurationNoticesItem.tsx

```diff
@@ -0,0 +1,192 @@
+    marginBottom: 20,
+    background: theme.palette.background.pageActiveAreaBackground,
+    padding: 12,
+    position: "relative",
+    paddingBottom: 24,
+  },
+  editButton: {
+    ...theme.typography.body2,
+    color: theme.palette.primary.main,
+    padding: "0px 16px",
+    fontSize: "16px",
+    position: "absolute",
+    right: 10,
+    top: 10,
+    textTransform: "uppercase",
+    "&:hover": {
+      opacity: 0.5,
+      },
+},
+  publishButton: {
+      ...theme.typography.body2,
+      color: theme.palette.primary.main,
+      padding: "0px 16px",
+      fontSize: "16px",
+      position: "absolute",
+      right: 10,
+      bottom: 10,
+      textTransform: "uppercase",
+      "&:hover": {
+        opacity: 0.5,
+      },
+  },
+  meta: {
+    "& > div": {
+      marginRight: 5,
+    },
+    position: "relative",
+    display: "flex",
+    flexWrap: "wrap",
+    alignItems: "center",
+    justifyContent: "space-between",
+    rowGap: "6px",
+    marginBottom: 8,
+    color: theme.palette.text.dim,
+    paddingTop: "0.6em",
+    marginRight: isFriendlyUI ? 40 : 20,
+
+    "& a:hover, & a:active": {
+      textDecoration: "none",
+      color: isFriendlyUI ? undefined : `${theme.palette.linkHover.dim} !important`,
+    },
+  },
+  postTitle: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    color: theme.palette.link.dim2,
+  },
+  username: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    fontWeight: 600,
+    marginLeft: 25,
+  },
+  rightGroup: {
+    display: "flex",
+    alignItems: "center",
+    gap: 10,
+  },
+  commentBody: {
+    ...commentBodyStyles(theme)
+  }
+});
+
+const { ContentItemBody, Button, BasicFormStyles, WrappedSmartForm } = Components;
+
+export const CurationNoticesItem = ({curationNotice, classes}: {
+  curationNotice: CurationNoticesFragment,
+  classes: ClassesType<typeof styles>
+}) => {
+  const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
+
+  const [ edit, setEdit ] = useState<boolean>(false)
+
+  const { create } = useCreate({
+    collectionName: "Comments",
+    fragmentName: 'CommentsList'
+  });
+
+  const {mutate: UpdateCurrentCurationNotice} = useUpdate({
+    collectionName: "CurationNotices",
+    fragmentName: 'CurationNoticesFragment',
+  });
+
+  const makeComment = async (curationNotice: CurationNoticesFragment
+  ) => {
+    if (!curationNotice.contents) {throw Error("Error creating comment")}
+
+    const comment = {
+      postId: curationNotice.postId,
+      userId: curationNotice.userId,
+      contents: { originalContents: { 
+        data: curationNotice.contents.originalContents.data,
+        type: curationNotice.contents.originalContents.type,
+      }} as EditableFieldContents
+    };
+
+    try {
+      const result = await create({data: comment});
+      const commentId = result.data?.createComment.data._id;
+      UpdateCurrentCurationNotice({
```

Comment:
I think that doing this as a two-step operation from the client will get us in trouble some day.  Probably this should actually be a single (custom) mutation, which performs both operations on the server.

Reply:
alas, i don't know how to begin to do that

i guess i have to go and add something to the mutations file

Reply:
we decided to leave this